lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -2890,14 +2890,14 @@ namespace Js
         typeHandler->Convert(webAssemblyObject, mode, 9);
         // Build wabt object
         Js::DynamicObject* wabtObject = library->CreateObject(true);
-        library->AddFunctionToLibraryObject(wabtObject, PropertyIds::convertWast2Wasm, &WabtInterface::EntryInfo::ConvertWast2Wasm, 2);
+        library->AddFunctionToLibraryObject(wabtObject, PropertyIds::convertWast2Wasm, &WabtInterface::EntryInfo::ConvertWast2Wasm, 1);
         library->AddMember(webAssemblyObject, PropertyIds::wabt, wabtObject, PropertyNone);
 #else
         typeHandler->Convert(webAssemblyObject, mode, 8);
 #endif
-        library->AddFunctionToLibraryObject(webAssemblyObject, PropertyIds::compile, &WebAssembly::EntryInfo::Compile, 2);
-        library->AddFunctionToLibraryObject(webAssemblyObject, PropertyIds::validate, &WebAssembly::EntryInfo::Validate, 2);
-        library->AddFunctionToLibraryObject(webAssemblyObject, PropertyIds::instantiate, &WebAssembly::EntryInfo::Instantiate, 2);
+        library->AddFunctionToLibraryObject(webAssemblyObject, PropertyIds::compile, &WebAssembly::EntryInfo::Compile, 1);
```

Comment:
Here I am not sure why it was length 2 and I couldn't find where in the spec it says it should be length 1, but the jsapi test was checking for `WebAssembly.compile.length === 1`